dbld/images/debian-stretch.dockerfile

```diff
@@ -1,9 +1,11 @@
 FROM debian:9
 LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, Balazs Scheidler <balazs.scheidler@oneidentity.com>"
+ENV OS_DISTRIBUTION=debian
+ENV RELEASE_CODE_NAME=9
```

Comment:
This will not work IMHO, code name shouldn't be the release number. Debian 9 code name is "stretch"

Reply:
Thanks, I fixed